servermon/hwdoc/models.py

```diff
@@ -86,7 +86,7 @@ class Project(models.Model):
 
     name = models.CharField(max_length=80)
     contacts = models.ManyToManyField(Person, through='Role')
-    comments = models.TextField(blank=True)
+    comments = models.TextField(blank=True, null=True)
```

Comment:
no, that's actually wrong. If you enable null here you will end up having problems because you have effectively 2 values for no input. '' and NULL. 

See more here https://docs.djangoproject.com/en/1.9/ref/models/fields/